src/javascript/app/pages/user/account/portfolio/portfolio.init.js

```diff
@@ -220,7 +220,20 @@ const PortfolioInit = (() => {
         is_initialized = false;
     };
 
+    const onReconnect = () => {
```

Comment:
is it different from calling onunload and then load? do we need to set a custom onreconnect for every page?

Reply:
There is no requirement for `onReconnect` on every page. If we need a page to do certain things when connection is back up, then we use this. if not, no command will be running.